code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,413 @@
+					to_chat(objective.owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+				else
+					var/list/objectivestoupdate
+					for(var/datum/mind/objective_owner in objective.get_owners())
+						to_chat(objective_owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+						for(var/datum/objective/update_target_objective in objective_owner.get_all_objectives())
+							LAZYADD(objectivestoupdate, update_target_objective)
+					objectivestoupdate += objective.team.objectives
+					for(var/datum/objective/update_objective in objectivestoupdate)
+						if(update_objective.target != old_target || !istype(update_objective,objective.type))
+							return
+						update_objective.target = objective.target
+						update_objective.update_explanation_text()
+						to_chat(objective.owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+						update_objective.owner.announce_objectives()
+				qdel(objective)
+
+// This function can not be undone; do not call this unless you are sure
+/obj/machinery/cryopod/proc/despawn_occupant()
+	var/mob/living/mob_occupant = occupant
+	var/list/crew_member = list()
+
+	crew_member["name"] = mob_occupant.real_name
+
+	if(mob_occupant.mind && mob_occupant.mind.assigned_role)
+		// Handle job slot/tater cleanup.
+		var/job = mob_occupant.mind.assigned_role
+		crew_member["job"] = job
+		SSjob.FreeRole(job)
+		if(LAZYLEN(mob_occupant.mind.objectives))
+			mob_occupant.mind.objectives.Cut()
+			mob_occupant.mind.special_role = null
+	else
+		crew_member["job"] = "N/A"
+	// Delete them from datacore.
+	var/announce_rank = null
+	for(var/datum/data/record/med_record in GLOB.data_core.medical)
+		if((med_record.fields["name"] == mob_occupant.real_name))
+			qdel(med_record)
+	for(var/datum/data/record/sec_record in GLOB.data_core.security)
+		if((sec_record.fields["name"] == mob_occupant.real_name))
+			qdel(sec_record)
+	for(var/datum/data/record/gen_record in GLOB.data_core.general)
+		if((gen_record.fields["name"] == mob_occupant.real_name))
+			announce_rank = gen_record.fields["rank"]
+			qdel(gen_record)
+
+	control_computer?.frozen_crew += list(crew_member)
+
+	// Make an announcement and log the person entering storage.
+	if(GLOB.announcement_systems.len)
+		var/obj/machinery/announcement_system/announcer = pick(GLOB.announcement_systems)
+		announcer.announce("CRYOSTORAGE", mob_occupant.real_name, announce_rank, list())
+		visible_message("<span class='notice'>\The [src] hums and hisses as it moves [mob_occupant.real_name] into storage.</span>")
+
+
+	for(var/obj/item/items in mob_occupant.GetAllContents())
```

Comment:
Change it to just `item`, this variable name is inaccurate as this is the individual item.